src/Data.py

```diff
@@ -22,12 +22,21 @@ def load_data_file(*args) -> dict:
 
     return filedata
 
-game_table = load_data_file('game.json')
-item_table = load_data_file('items.json')
-location_table = load_data_file('locations.json')
-region_table = load_data_file('regions.json')
-category_table = load_data_file('categories.json') or {}
-meta_table = load_data_file('meta.json') or {}
+def convert_to_list(data, property_name: str) -> list:
+    if isinstance(data, dict):
+        data = data.get(property_name, [])
+    return data
+
+game_table = load_data_file('game.json') #dict
+item_table = convert_to_list(load_data_file('items.json'), 'data') #list
+location_table = convert_to_list(load_data_file('locations.json'), 'data') #list
+region_table = load_data_file('regions.json') #dict
+category_table = load_data_file('categories.json') or {} #dict
+meta_table = load_data_file('meta.json') or {} #dict
+
+# Removal of schemas in root of tables
+region_table.pop('$schema', '')
+category_table.pop('$schema', '')

```

Comment:
This shouldn't be necessary.   
And in terms of micro-optimizations, I'm more concerned by load time than a few bytes of memory.

Reply:
its not for optimization we loop thru both as if they are real region/category elsewhere in the code

Reply:
for example in create_regions the code think $schema is a real region and try to create it.
in option.py the category get looped and call .get on $schema which is a str not a dict and thus dies

Reply:
its either a pop() here or add a check everywhere that needs it to skip if str both is valid

Reply:
Ah, true. I'm happy with the pop.